en/intro/conventions.rst

```diff
@@ -156,6 +156,14 @@ Now that you've been introduced to CakePHP's fundamentals, you might try a run
 through the :doc:`/tutorials-and-examples/cms/installation` to see how things fit
 together.
 
+Plugins Conventions
+===================
+It is useful to prefix a CakePHP plugin with "cakephp-" in the package name.
+This makes the name semantically related on the framework it depends on.
+Ideally, this should be a prefix dependencies of 'cakephp' as the basis.
+Do **not** use the CakePHP namespace (cakephp) as this is reserved to CakePHP owned plugins.
```

Comment:
I would rather link to the awesome list conventions, as you are duplicating the content here now isolated from that list.

Reply:
I'm already a little confused. Please, can you edit it yourself the way you see it right? Or write completely here in the commentary so that I make a complete change. If it is necessary at all, because I see that not everyone agrees and I do not quite understand what to do next.

Reply:
>because I see that not everyone agrees 

my mean of above text is 
this is not a cake convention but is cake recommendation
and if @dereuromark  was agree , we can change `Plugins Conventions` to something like `Plugins Recommendation`

Reply:
Isn't a convention a recommendation? Just mildly more opinionated (for good reason)?